lib/ansible/plugins/connection/ssh.py

```diff
@@ -562,6 +573,10 @@ def _build_command(self, binary, *other_args):
             self.sshpass_pipe = os.pipe()
             b_command += [b'sshpass', b'-d' + to_bytes(self.sshpass_pipe[0], nonstring='simplerepr', errors='surrogate_or_strict')]
 
+            password_prompt = self.get_option('sshpass_prompt')
+            if password_prompt:
```

Comment:
is not None, since it default to null, though you might want to treat '' specifically or make that the default.

Reply:
Made `''` the default.